docs/source/articles/decorate-and-write-tests.html.md

```diff
@@ -5,3 +5,38 @@ excerpt: TODO
 ---
 
 # Decorate & Write Tests
+
+
+## Decorate a Workarea Test
+
+Does the decorator generator also decorate the tests for the class you're decorating?
```

Comment:
No, but you should probably just write a new test class because decorating tests unnecessarily gets weird with traces and commands to run and whatnot. Easier to write a new test class.